queries.sql

```diff
@@ -15,4 +15,17 @@ SELECT name FROM animals WHERE neutered = true;
 /*  - Find all animals not named Gabumon. */
 SELECT * FROM animals WHERE name != 'Gabumon';
 /*  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
-SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+/*   - How many animals are there? */
+SELECT COUNT(*) FROM animals;
+/*   - How many animals have never tried to escape? */
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+/*   - What is the average weight of animals? */
+SELECT AVG(weight_kg) FROM animals;
+/*   - Who escapes the most, neutered or not neutered animals? */
+SELECT SUM(escape_attempts) FROM animals GROUP BY neutered;
+SELECT AVG(escape_attempts) FROM animals GROUP BY neutered;
+/*   - What is the minimum and maximum weight of each type of animal? */
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+/*   - What is the average number of escape attempts per animal type of those born between 1990 and 2000? */
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31' GROUP BY species;
```

Comment:
- [ ] According to the project requirement, they should be some transactions for example 
```
 BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```
kindly add all the transactions required